src/utils/error.ts

```diff
@@ -224,6 +224,23 @@ export class FirebaseInstanceIdError extends FirebaseError {
   constructor(info: ErrorInfo, message?: string) {
     // Override default message if custom message provided.
     super({ code: 'instance-id/' + info.code, message: message || info.message });
+    (this as any).__proto__ = FirebaseInstanceIdError.prototype;
+  }
+}
+
+/**
+ * Firebase Installations service error code structure. This extends FirebaseError.
```

Comment:
Should Installations and FirebaseError be backticked?

Reply:
I think it makes sense to add markdown around FirebaseError. Firebase Installations service is meant as a literal here.